internal/iam/command_rbac_role_binding_create.go

```diff
@@ -49,6 +49,14 @@ func (c *roleBindingCommand) newCreateCommand() *cobra.Command {
 				Text: `Grant the "ResourceOwner" role to principal "User:u-123456" and subject "test" in schema context "schema_context" for Schema Registry "lsrc-123456" in the environment "env-12345":`,
 				Code: `confluent iam rbac role-binding create --principal User:u-123456 --role ResourceOwner --environment env-12345 --schema-registry-cluster lsrc-123456 --resource "Subject::.schema_context:test"`,
 			},
+			examples.Example{
+				Text: `Grant the role "FlinkDeveloper" to the principal "User:u-123456" in the environment "env-12345":`,
+				Code: "confluent iam rbac role-binding create --principal User:u-123456 --role FlinkDeveloper --environment env-12345",
+			},
+			examples.Example{
+				Text: `Grant the role "FlinkDeveloper" to the principal "User:u-123456", in the environment "env-12345" for the compute pool "lfcp-123456" in the Flink region "us-east-2":`,
+				Code: "confluent iam rbac role-binding create --principal User:u-123456 --role FlinkDeveloper --environment env-12345 --flink-region aws.us-east-2 --resource compute-pool:lfcp-123456",
```

Comment:
Can we update the `compute-pool` resource type to `ComputePool`? All the other resourcetypes seem to imply CamelCase (e.g. `Topic`, `Subject`, `TransactionId`)

Reply:
sure, will do

Reply:
It's not updated on my end?

Reply:
fixin this now